### ISSUE TITLE
Accuracy is only 70%

### ISSUE THREAD

**Issue by Achilles-96** [jiyfeng/tfkld#3]
I put dev data to be last 815 lines of train data, as you mentioned in the paper that you used 20%. I am only getting 70% accuracy. Can you tell if I should make any changes.

**Comment by SimonOst** [comment]
I'm facing the very same problem. Do I get it correctly that the input data should have the following format:
(0|1)\tTEXT1\tTEXT2
I was wondering if I'm using the wrong format maybe... @Achilles-96 , which format did you use?

**Comment by Achilles-96** [comment]
https://github.com/jiyfeng/tfkld/blob/master/python/weight.py#L24-L26
That format is correct, as you can see in code.

**Comment by mlberlin** [comment]
Given the definition of the confusion matrix used by scikit learn, see http://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html the calculation of precision and recall (and thus F1) in classification.py seems incorrect.  Intead of
~~~~
   p = (1.0 * mat[0,0] / mat[:,0].sum()) + (1.0 * mat[1,1] / mat[:,1].sum())
   p = p / 2.0
~~~~
it should read
~~~~
   p = 1.0 * mat[1,1] / mat[:,1].sum()
~~~~
and instead of
~~~~
   r = (1.0 * mat[0,0] / mat[0,:].sum()) + (1.0 * mat[1,1] / mat[1,:].sum())
   r = r / 2.0
~~~~
it should read
~~~~
   r = 1.0 * mat[1,1] / mat[1,:].sum()
~~~~
according to that definition.
